src/hugolib/aasa_test.go

```diff
@@ -18,26 +18,43 @@ func TestAASAOutput(t *testing.T) {
 	testCases := []struct {
 		prefix  string
 		bundle  string
+		version string
 		passing bool
 	}{
 		{
 			"ABCDE12345",
 			"com.gothamhq.ios",
+			"",
```

Comment:
I don't think an empty string should be the value for the v2 schema when "v1" is the value for the v1 schema. We should use "v2" for the v2 schema to be consistent and explicit.